vectordraw.js

```diff
@@ -1,6 +1,25 @@
 'use strict';
 
 var VectorDraw = function(element_id, settings) {
+    this.board = null;
+    this.dragged_vector = null;
+    this.drawMode = false;
+    this.history_stack = {undo: [], redo: []};
+    this.settings = this.sanitiseSettings(settings);
```

Comment:
@smarnach nit: not technically a typo, but other parts of codebase use "American" spelling of -ze, not -se.